HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentsController.swift

```diff
@@ -21,6 +21,7 @@ protocol PaymentComponentsProtocol {
     var selectedPaymentProvider: PaymentProvider? { get set }
     func loadPaymentProviders()
     func checkIfDocumentIsPayable(docId: String, completion: @escaping (Result<Bool, GiniHealthError>) -> Void)
+    func checkIfDocumentContainsMultipleInvoices(docId: String, completion: @escaping (Result<Bool, GiniHealthError>) -> Void)
```

Comment:
[Clarification]: Why do we also add checkIfDocumentContainsMultipleInvoices(docId) in `PaymentComponentsProtocol`?

Reply:
Beside checking this value in the extraction in the task is specified to have an extra method in Protocol: 
` Additional method in the SDK to inform the user that the document contains multiple invoices.`

Reply:
yes, we have it in the SDK do we need this method in the PaymentComponentsProtocol?

Reply:
Yes. We can remove it, but then they need to create an extra GiniHealth SDK object in order to access this method. Because in PaymentComponentController, we have a private GiniHealth object.